Libraries/LibWeb/HTML/HTMLMetaElement.cpp

```diff
@@ -143,6 +146,41 @@ void HTMLMetaElement::inserted()
             document().set_pragma_set_default_language(language);
             break;
         }
+        case HttpEquivAttributeState::ContentSecurityPolicy: {
+            // https://html.spec.whatwg.org/multipage/semantics.html#attr-meta-http-equiv-content-security-policy
+            // This pragma enforces a Content Security Policy on a Document. [CSP]
+            // 1. If the meta element is not a child of a head element, return.
+            if (!is<HTMLHeadElement>(parent()))
+                break;
+
+            // 2. If the meta element has no content attribute, or if that attribute's value is the empty string, then return.
+            if (!has_attribute(AttributeNames::content))
```

Comment:
This ends up doing duplicate work with the one below one since get_attribute_value falls back to empty string when not found. I suppose we can either get rid of this line, or do a `get_attribute` instead if we want to explicitly check both.